com.unity.render-pipelines.universal/Shaders/Utils/Deferred.hlsl

```diff
@@ -73,6 +74,9 @@ Light UnityLightFromPunctualLightDataAndWorldSpacePosition(PunctualLightData pun
     {
         light.shadowAttenuation = AdditionalLightShadow(shadowLightIndex, positionWS, lightDirection, shadowMask, punctualLightData.occlusionProbeInfo);
     }
+
+    light.lightLayers = punctualLightData.lightLayers;
```

Comment:
```suggestion
    light.layerMask = punctualLightData.layerMask;
```

This is what I get confused when using lightLayer vs layer mask

Reply:
Ok, I renamed (a lot of code) to layerMask.